chain/client/src/client.rs

```diff
@@ -1772,8 +1778,23 @@ impl Client {
             Some(shard_chunk.clone()),
             self.chain.mut_chain_store(),
         )?;
+
+        let chunk_header = encoded_chunk.cloned_header();
+        if let Some(chunk_distribution) = &self.chunk_distribution_network {
+            if chunk_distribution.config.enabled {
+                let partial_chunk = partial_chunk.clone();
+                let mut thread_local_client = chunk_distribution.clone();
+                near_performance_metrics::actix::spawn("ChunkDistributionNetwork", async move {
+                    use crate::chunk_distribution_network::ChunkDistributionClient;
```

Comment:
nit: I guess there is no harm in having this `use` at the top of the file, this seems to be more conventional in nearcore codebase

Reply:
Addressed as part of https://github.com/near/nearcore/pull/10480/commits/2a4d18189d23cc4a4a8d72b248ac65b8d87006e0